docs/features/autoupdater.rst

```diff
@@ -16,10 +16,16 @@ be set on the command line, or it can be taken from the ``site.mk``.
 
 The priority defines the maximum number of days that may pass between releasing an update and installation
 of the images. The update probability will start at 0 after the release time mentioned in the manifest
+
+>  where and how is this mentioned?
```

Comment:
I am sorry, but this is a *pull request*, not a madhouse! This is no enhancement of an explanation, but a comment that should go into an issue, not a pull request.